src/DeployToAzure/DeployToAzure.csproj

```diff
@@ -1,125 +1,44 @@
     <Reference Include="System.Core" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.IO.FileSystem.Primitives, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\System.IO.FileSystem.Primitives.4.3.0\lib\net46\System.IO.FileSystem.Primitives.dll</HintPath>
-    </Reference>
-    <Reference Include="System.IO.Packaging, Version=4.0.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\System.IO.Packaging.4.5.0\lib\net46\System.IO.Packaging.dll</HintPath>
-    </Reference>
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
-    <Reference Include="WindowsBase" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Annotations.cs" />
-    <Compile Include="ConfigurationParser.cs" />
-    <Compile Include="DisposeTracker.cs" />
-    <Compile Include="Management\AzureDeploymentCheckOutcome.cs" />
-    <Compile Include="Management\AzureDeploymentSlot.cs" />
-    <Compile Include="Management\AzureManagementApiWithRetries.cs" />
-    <Compile Include="Management\AzureManagementLowLevelApi.cs" />
-    <Compile Include="Management\BadRequestException.cs" />
-    <Compile Include="Management\DeploymentConfiguration.cs" />
-    <Compile Include="Management\DeploymentSlotUri.cs" />
-    <Compile Include="Management\Http.cs" />
-    <Compile Include="Management\IAzureManagementLowLevelApi.cs" />
-    <Compile Include="Management\RequestUri.cs" />
-    <Compile Include="PackageManifest.cs" />
-    <Compile Include="PackagingExtensions.cs" />
-    <Compile Include="PartStream.cs" />
-    <Compile Include="Program.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="ServiceDefinition.cs" />
-    <Compile Include="Utility\OurConsoleTraceListener.cs" />
-    <Compile Include="Utility\DisposeAction.cs" />
-    <Compile Include="Utility\DynamicXml.cs" />
-    <Compile Include="Utility\ExceptionExtensions.cs" />
-    <Compile Include="Utility\ExpressionHelpers.cs" />
-    <Compile Include="Utility\FailFast.cs" />
-    <Compile Include="Utility\RetryLogic.cs" />
-    <Compile Include="Utility\SpinLoop.cs" />
-    <Compile Include="Utility\OurTrace.cs" />
-    <Compile Include="Utility\TraceSourceExtensions.cs" />
-    <Compile Include="VMSizeChanger.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
     <None Include="DeployToAzure.ps1">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Include="packages.config" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
+  <Target Name="ILMerge">
```

Comment:
instead of "OneBuild" build this plays a role in the `build.cmd` `dotnet build ... /t:ILMerge` using pattern matching for the .dlls.